internal/testbed/integration/statsd/testdata/statsd/job.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: statsd-generator-job
+  namespace: e2estatsd
+spec:
+  template:
+    spec:
+      containers:
+      - name: netcat
+        image: alpine
+        command: ["/bin/sh", "-c"]
+        args: [
+          "echo 'test.metric:42|g|#myKey:myVal' | nc -u -w 1 -v otelcolsvc.e2estatsd.svc.cluster.local 8125 && \
+           sleep 1 && \
+           echo 'timerMetric:320|ms|@0.1|#timerKey:timerVal' | nc -u -w 1 -v otelcolsvc.e2estatsd.svc.cluster.local 8125"
```

Comment:
Sorry I'm not familiar with StatsD but I just wanted to make sure: does this metric ends up "touching"/testing the processor config where we convert it to a histogram, prior to exporting it as a gauge?

Reply:
yes it does. the `ms` means it is a timer. There are 3 types of statsd metrics which the config converts to histograms and timer is one of them. This particular entry means the value of `320` should be entered into the histogram 10 times (`@0.1` means it is sampled at a rate of 10%).

Reply:
>prior to exporting it as a gauge
it should not be exported as a gauge. it should be converted to histogram and that's it